examples/test/multithreading/test_i_raw_aes_keyring_multithreaded_example.py

```diff
@@ -0,0 +1,34 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Test suite for the Raw AES keyring example with multi-threading."""
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+import pytest
+
+import aws_encryption_sdk
+from aws_encryption_sdk import CommitmentPolicy
+
+from ...src.multithreading import encrypt_and_decrypt_with_keyring
+from ...src.multithreading.raw_aes_keyring import create_keyring
+
+pytestmark = [pytest.mark.examples]
+
+
+def test_encrypt_and_decrypt_with_keyring(n_threads=10):
```

Comment:
Let's consider parametrizing this across `n_threads` and `test_duration`. As-is, testing this with 10 threads once is a very lightweight test.

I'd suggest

`n_threads = [4, 16, 64]`
`duration (seconds) = [2, 10, 60]`